src/core/sagas/userInfoSagas.js

```diff
@@ -0,0 +1,24 @@
+import { takeEvery, put } from 'redux-saga/effects';
+
+import { token } from "../token";
+import { USER_INFO_ACTION, userInfoAction } from "../actions/userInfoAction";
+
+export default takeEvery(USER_INFO_ACTION.REQUEST, function*(action) {
+    const domainName = "http://dev.ripple.foundation";
```

Comment:
This should come from a configuration, e.g. `window.config`. 

See https://immutablewebapps.org/

Reply:
Done. Domain name was moved to configuration file